src/dbcp/extract/acp_projects.py

```diff
@@ -5,6 +5,25 @@
 import pandas as pd
 
 import dbcp
+import dbcp.extract
+import dbcp.extract.helpers
+
+
+def _extract_acp_projects_concatenation() -> pd.DataFrame:
+    """Extract a dataframe with all ACP quarters concatenated together.
+
+    Used for creating the ACP changelog.
+    """
+    file_paths = dbcp.extract.helpers.cache_gcs_archive_bucket_contents_locally(
+        gcs_dir_name="acp"
+    )
+    concat_df = pd.DataFrame()
+    for path in file_paths:
+        new_df = pd.read_csv(path)
+        date = pd.to_datetime("-".join(path.parts[-1].split(".")[0].split("_")[1:]))
+        new_df["report_date"] = date
+        concat_df = pd.concat([concat_df, new_df])
+    return concat_df
```

Comment:
This reads in all the raw ACP data and concatenates them together in preparation for making a changelog. I could update this to check whether there's already a change log in the data warehouse, and then just extract the latest version of the data to compare against the existing change log. However, then we'd have to know the date of the last version of the data that the existing change log was created from.